tools/quickstart/quickstart/bridge.py

```diff
@@ -95,3 +112,31 @@ def _show_file_override_dialog(base_dir, bridge_config_file):
             )
         else:
             assert False, "unreachable"
+
+
+def read_json_rpc_url():
+    url = click.prompt("JSON RPC url")
+    if verify_url(url):
+        return url
+    else:
+        url = click.prompt(
+            f"The given url seems invalid {url}, please correct it or enter the same url if valid"
```

Comment:
Thats is weird, just ask the user again? why should it succeed the second time?

Reply:
That is how I understood the requirement: we try to validate the URL, if it looks wrong we warn the user but let him proceed anyway. 

The way to let him proceed could be to ask him whether the URL is correct and let him change it if there was just a mistake, or to directly let him retype it. I chose the latter.

Reply:
I thought this was about making the test call to the rpc endpoint, which is completely missing. 
I dont get you workflow, so if I misstyped it a second time I am doomed?